cob_image_flip/ros/src/image_flip.cpp

```diff
@@ -478,14 +545,14 @@ void ImageFlip::pcCallback(const sensor_msgs::PointCloud2::ConstPtr& point_cloud
 	cv::Mat rot33 = cv::Mat::eye(3,3,CV_64FC1);
 	for (int r=0; r<2; ++r)
 		for (int c=0; c<3; ++c)
-			rot33.at<double>(r,c) = rot_mat.at<double>(r,c);
+            atForDiffImTypes(rot33, r, c, atForDiffImTypesReturn(rot_mat, r, c));
 	cv::Mat rot33_inv = rot33.inv();
 	cob_perception_msgs::Float64ArrayStamped rot33_inv_msg;
 	rot33_inv_msg.header = point_cloud_msg->header;
 	rot33_inv_msg.data.resize(9);
 	for (int r=0; r<3; ++r)
 		for (int c=0; c<3; ++c)
-			rot33_inv_msg.data[r*3+c] = rot33_inv.at<double>(r,c);
```

Comment:
Just leave it the old way in this case, because the rotation matrix is always the same format CV_64FC1.